R/geom-violin.r

```diff
@@ -107,7 +110,19 @@ GeomViolin <- proto2(
       # Needed for coord_polar and such
       newdata <- rbind(newdata, newdata[1,])
 
-      ggname(self$my_name(), GeomPolygon$draw(newdata, ...))
+      if (any(data$is.quantile)) {
```

Comment:
Do you mean *is.quantile* vector should be passed as a parameter to the draw function? Or do you mean that the *quantiles* desired should be passed? For the latter, I placed it in the *ydensity* stat because I don't believe the raw data can be accessed from the *geom-violin* draw function. Or can they?

If there is existing code that does what you are referring to and can act as a template for me, please point the way.

thx
dan